Client/src/Components/Inputs/Field/index.jsx

```diff
@@ -57,7 +57,7 @@ const Field = forwardRef(
 		return (
 			<Stack
 				gap={theme.spacing(2)}
-				className={`${className ?? `field field-${type}`}`}
+				className={`field field-${type} ${className}`}
```

Comment:
@jennifer-gan I'm really glad to see you picked up on this issue too!  I just submitted a PR that does basically this, could you remove this from this PR so that this one is focused on the details page please?  One feature or fix per PR :smile: